identity/client/src/pages/users/modals/isValidEmail.tsx

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+export function isValidEmail(email: string): boolean {
+  return /^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/.test(
```

Comment:
💭 We also perform validation of the email in the API layer, there we rely on `commons-validator` to provide the check of a valid email, does this regex here align with that? I think this is likely also pretty well defined by spec so not a great concern for me here just wanted to make sure we were aligning the UI with API

Reply:
I've just checked locally and a string of `bar@foo` passes the UI check but fails in the API (I also don't see the error in the UI for that)

Reply:
I updated the regex to use a bit stricter email validation. I want to keep things simple in the front-end, since we already have a backend validation. The validation is to give a user immediate feedback.
We already have an issue for the "API failures not working in the front-end". Once this is sorted, users will see the backend errror as well.